core/sql/optimizer/NormRelExpr.cpp

```diff
@@ -7050,8 +7077,8 @@ void RelRoot::recomputeOuterReferences()
       child(0)->getGroupAttr()->getCharacteristicInputs().
                   getLeafValuesForCoverTest(leafValues, emptyGA, emptySet);
       CMPASSERT((getGroupAttr()->getCharacteristicInputs().contains
-                (child(0)->getGroupAttr()->getCharacteristicInputs())) || 
-                (getGroupAttr()->getCharacteristicInputs().contains (leafValues)));
+		 (child(0)->getGroupAttr()->getCharacteristicInputs())) || 
```

Comment:
Will fix my TABs, likely after calling you. I use emacs and am not sure how to set this correctly.